core/App/screens/CredentialOffer.tsx

```diff
@@ -62,8 +63,10 @@ const CredentialOffer: React.FC<CredentialOfferProps> = ({ navigation, route })
   const handleAcceptPress = async () => {
     try {
       setAcceptModalVisible(true)
-
-      await agent.credentials.acceptOffer(credential.id)
+      await agent.credentials.acceptOffer({
+        credentialRecordId: credential.id,
+        autoAcceptCredential: AutoAcceptCredential.Always,
```

Comment:
Always means it will accept subsequent messages no matter the content. I think ContentApproved may be a better fit here